integration_tests/security-providers/build.gradle

```diff
@@ -4,12 +4,11 @@ apply plugin: RoboJavaModulePlugin
 
 dependencies {
     api project(":robolectric")
-    api "junit:junit:${junitVersion}"
+    api libs.junit
     compileOnly AndroidSdk.MAX_SDK.coordinates
 
     testRuntimeOnly AndroidSdk.MAX_SDK.coordinates
-    testImplementation "com.google.truth:truth:${truthVersion}"
-    testImplementation "org.conscrypt:conscrypt-openjdk-uber:2.4.0"
-    testImplementation "com.squareup.okhttp3:okhttp"
-    testImplementation platform("com.squareup.okhttp3:okhttp-bom:4.8.0")
+    testImplementation libs.google.truth
+    testImplementation libs.conscrypt
+    testImplementation libs.okhttp
```

Comment:
Maybe we need keep `platform(libs.okhttp)` too: https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.dsl.DependencyHandler.html#org.gradle.api.artifacts.dsl.DependencyHandler:platform(java.lang.Object).